### PR TITLE
Add prepare-release Tekton job

### PR DESCRIPTION
**Pull request by vfreex**
This PR adds `prepare-release` Tekton pipeline.

Currently we are keeping the Tekton pipelines and related Dockerfiles in this repo
so that we can use Doozer and Elliott versions referenced by the submodules.
In the future, we will move Tekton pipelines to a separate repo.

Test this job by running

```sh
tkn -n art-cd pipeline start prepare-release
```

See https://coreos.slack.com/archives/GDBRP5YJH/p1612454005134700 for detailed instructions to try this job.

### CODE REVIEW

File: tekton-pipelines/images/artcd/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  koji \
+  rhpkg
+# install elliott, doozer, and pyartcd
+COPY ./art-tools/ /usr/local/src/
+COPY ./pyartcd/ /usr/local/src/pyartcd/
+RUN pushd /usr/local/src/ \
+  && pip3 wheel --wheel-dir /usr/local/src/wheels --use-deprecated=legacy-resolver --no-deps ./elliott ./doozer ./pyartcd
+RUN python3 -m pip install --ignore-installed /usr/local/src/wheels/*.whl \
+  # make Python trust system CA certificates
+  && python3 -m pip install certifi && ln -sf /etc/pki/tls/cert.pem /usr/local/lib/python3.6/site-packages/certifi/cacert.pem
```
Comment (Ximinhan) [inline tekton-pipelines/images/artcd/Dockerfile:37]: elliott verify-payload need oc command, so in base image need to add https://mirror.openshift.com/pub/openshift-v4/clients/oc/latest/linux/oc.tar.gz or https://downloads-openshift-console.apps.ocp4.prod.psi.redhat.com/amd64/linux/oc.tar into /usr/bin/

Comment (Ximinhan) [reply tekton-pipelines/images/artcd/Dockerfile:37]: https://console-openshift-console.apps.ocp4.prod.psi.redhat.com/k8s/ns/art-cd/tekton.dev~v1beta1~PipelineRun/prepare-release-m1zltk/logs

Comment (vfreex) [reply tekton-pipelines/images/artcd/Dockerfile:44]: Done.

**Review by thiagoalessio — APPROVED** [review]
/lgtm

**Pull request merged**